app/scripts/services/TextMessageMessage.svc.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+angular.module('yourStlCourts').factory('TextMessageMessage', function ($resource, $q,$rootScope) {
```

Comment:
TextMessageMessage as a name is not easily digested into understanding. How about some consistency with everywhere else like "SMSMessage" ?